libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/insightOnDashboard.spec.ts

```diff
@@ -79,3 +84,106 @@ describe("Insights on dashboard", () => {
         },
     );
 });
+
+describe("Date filtering on Insight", () => {
+    beforeEach(() => {
+        Navigation.visit("dashboard/dashboard-date-filtering-on-insight-scenario");
+        editMode.isInEditMode(false).edit().isInEditMode();
+    });
+
+    it(
+        "Remember Last Setting After Selecting Another Insight",
+        { tags: ["checklist_integrated_tiger"] },
+        () => {
+            new Widget(0).waitChartLoaded();
+
+            const widgetConfiguration = new WidgetConfiguration(0).open().openConfiguration();
+            widgetConfiguration.selectDateDataset(DATASET_CREATED);
+
+            new Widget(1).hasTitle("Column with two measures by date");
+
+            widgetConfiguration.hasDatasetSelected(DATASET_CREATED);
```

Comment:
check again in Graphene test, does it check the setting of the first widget after select the second one, or does it check the setting of the second widget to make sure it has the same setting as the first widget?

Your test seems not meet any of above 2 cases:
Your test flow:
1. select widget0 -> open configuration --> select Dataset_created
2. check widget1 title 
3. verify again widget0 date configuration = created

Reply:
how can it check date dimension of widget 1 while configuration of widget2 is opening?